### ISSUE TITLE
html documents served are downloaded

### ISSUE THREAD

**Issue by LordUber** [http-party/http-server#623]
http-server example on this page
https://stackabuse.com/how-to-start-a-node-server-examples-with-the-most-popular-frameworks/
**Tell us about your environment**
- **exact http-server version:** 0.12.1
- **Node version:** 12.16.2
- **Platform:** Win10

**Other information (related issues, suggestions for a fix, etc):**
Just learning about the server but when it goes to serve the example page it downloads it instead of displaying it. Does this in both Chrome and Firefox (latest versions). Any chance you could explain what I am doing wrong if this is my issue? Thanks.

**Comment by georgem3** [comment]
I can at least confirm the Firefox behavior, it's working on chrome (latest)

**Comment by thornjad** [comment]
What does the file you're trying to serve look like? Does it have the file extension `.html`? Do other servers serve it properly?

**Comment by LordUber** [comment]
Thanks for your reply. This is happening on both Chrome and Firefox.
This is the code of index.html. It was originally a visual studio code template, but I added to it to see if that helped.

<!DOCTYPE html xmlns="http://www.w3.org/1999/xhtml"><html>  <head>  <meta http-equiv="content-type" content="text/html; charset=UTF-8">  <meta charset="utf-8">  <head>    <title>Hello from http-server</title>  </head>  <body>
    <h1>Hello from http-server</h1>

**Comment by LordUber** [comment]
[index.docx](https://github.com/http-party/http-server/files/4517350/index.docx)

**Comment by thornjad** [comment]
Is the file actually called `index.docx`? If so, that makes it a Microsoft Word document, which should be downloaded. There isn't really any way for a regular web server to serve a file like that.

**Comment by LordUber** [comment]
No, I just put it into a docx file so you can see it - I couldn't upload
the html file.

Thanks

**Comment by Joao-Pedro-S** [comment]
This is probably a problem with the http response header. Maybe Chrome fixes it, but firefox doesn`t

**Comment by zbynek** [comment]
Seems like a duplicate of https://github.com/http-party/http-server/issues/657 . A complete list of response headers might help debugging this.
